arcsi/api/show.py

```diff
@@ -64,19 +62,27 @@ def make_show(self, data, **kwargs):
         return Show(**data)
 
 
-show_details_schema = ShowDetailsSchema()
-show_details_partial_schema = ShowDetailsSchema(partial=True)
-many_show_details_schema = ShowDetailsSchema(many=True)
-many_show_details_basic_schema = ShowDetailsSchema(many=True, 
-                                                   only=("id", "active", "name", "description",
-                                                         "week", "day", "start", "end",
-                                                         "cover_image_url", "archive_lahmastore_base_url"))
+show_schema = ShowDetailsSchema()
+show_archive_schema = ShowDetailsSchema(only=("name", "cover_image_url", 
+                                                    "day", "start", "end",
+                                                    "frequency", "language",
+                                                    "active", "description",
+                                                    "items"))
+show_partial_schema = ShowDetailsSchema(partial=True)
+shows_schema = ShowDetailsSchema(many=True)
+shows_schedule_schema = ShowDetailsSchema(many=True, 
+                                                   only=("active", "name", "cover_image_url",
+                                                         "day", "start", "end",
+                                                         "description", "archive_lahmastore_base_url"))
+shows_archive_schema = ShowDetailsSchema(many=True, 
```

Comment:
One small note here: Is this `archive` schema and the `/show/archive` endpoint for the front-end Shows list page? Maybe I misunderstand but in that case maybe we could rename it.